.github/workflows/ci.yml

```
@@ -58,6 +58,15 @@ jobs:
           sleep 5
         working-directory: ${{env.BOX_DIR}}
 
+      - name: Boot Fedora 34, 35 into latest kernel
+        if: "${{ matrix.distro == 'fedora34' || matrix.distro == 'fedora35' }}"
```

Let's boot just Fedora 35 to the newest kernel to check our module on different kernel versions.
```suggestion
        if: "${{ matrix.distro == 'fedora35' }}"
```